friends/src/actions/Index.js

```diff
@@ -0,0 +1,66 @@
+import axios from 'axios';
```

Comment:
This file needs to be named with a lowercase "i"

```bash
Failed to compile.

./src/components/App.js
Module not found: `/Users/mixelpix/Downloads/Redux-II-master/friends/src/actions/index.js`
does not match the corresponding path on disk `Index.js`.
```

Reply:
The Index.js file isn’t supposed to be in the commits. It’s the one I tried to delete when you were helping me. The correct one is the lower case index.js